README.md

```diff
@@ -89,10 +89,14 @@ If you want to report a bug, first, thanks a lot. To help us, please indicate yo
 
 - Fork and clone
 - Master is usually unstable, checkout to a tag to have a stable state of the app
-- `npm install && npm run dev`
-- Enable dev mode in the app in the settings view to show DevTools
 
-Webpack will watch for JSX and SASS changes.
+- Install the latest version of electron by running `npm install -g electron`
+- NPM will install electron globally. You can use it on every of your electron projects now.

```

Comment:
I'd put the manual electron install too. I don't like to use global package for that.

My recommended way is to download Electron, then place the app in `resources/app`.